src/internal/connector/data_collections.go

```diff
@@ -75,16 +75,18 @@ func (gc *GraphConnector) ProduceBackupCollections(
 		path.ServiceType(sels.Service),
 		sels.DiscreteOwner)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, false, err
 	}
 
 	if !serviceEnabled {
-		return []data.BackupCollection{}, nil, nil
+		// TODO(meain): How would this work? Should we try pulling to decide?
+		return []data.BackupCollection{}, nil, false, nil
```

Comment:
TODO: Can we safely assume there will be no previous backup if the service is not enabled?